### DISCUSSION TITLE
DPI scaling not working

### DISCUSSION THREAD

**Discussion by Karteek649** [dotnet/wpf#6429] [Q&A]
I created WPF .Net Framework dll and exposed through COM. And I tried to invoke the WPF UI from win32 application where the DPI awareness is set to PerMonitorV2 but when I change the Scaling the DPI is not working on WPF window which is inside the COM dll. 

But when same dll invoked from Another WPF application or .NET Framework project the scaling is working fine for WPF window in COM dll. 

Is there any issue in DPI scaling if WPF exposed by COM? 

Also when I make the win32 app PerMonitor or PerMonitorV2 and trying to invoke UI in WPF dll using COM the WPF window is not scaling acccording to DPI change. But same is working when I make Win32 app system aware.

**Comment by jeffstall** [comment]
Hi Karteek649,
There isn't enough information to understand the question. Can you please create a minimal test app or minimal source code to reproduce the problem? We can then try to help identify what is not working properly.
Thanks!
